CourseApp.Tests/DemoTest.cs

```diff
@@ -10,5 +10,76 @@ public void Test1()
         {
             Assert.True(true);
         }
+
+        [Fact]
+        public void TestFunction1()
```

Comment:
Please add name without number and use Inline data for the test with different parameters